api/repository/tasks.rb

```diff
@@ -0,0 +1,12 @@
+module Huertask
```

Comment:
wtf? this must be a file from another PR :P Please review the changes before committing